packages/cli/src/ui/components/messages/ToolMessage.tsx

```diff
@@ -90,9 +95,15 @@ export const ToolMessage: React.FC<ToolMessageProps> = ({
           description={description}
           emphasis={emphasis}
         />
-        {isThisShellFocused && (
+        {isThisShellFocusable && (
           <Box marginLeft={1}>
-            <Text color={theme.text.accent}>[Focused]</Text>
+            <Text
+              color={
+                isThisShellFocused ? theme.text.accent : theme.text.secondary
+              }
+            >
+              {isThisShellFocused ? '(Focused)' : '(ctrl+f to focus)'}
```

Comment:
I think having the `(ctrl+f to focus)` in accent color as well makes it more visible and will result in more users figuring it out.

@galz10 I also wonder if we should make `(Focused)` become `(focused, ctrl+f to unfocus)`

Reply:
i feel like `(focused, ctrl+f to unfocus)` would be too crowded but i can make that change.

Reply:
I'll leave it up to you! 👍 But i like the accent color for both 😄